File: 1295-find-numbers-with-even-number-of-digits/1295-find-numbers-with-even-number-of-digits.py
class Solution(object):
    def findNumbers(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        evendc=0
        for i in nums:
            l=len(str(i))
            if l%2==0:
                evendc+=1
        return evendc

File: 1295-find-numbers-with-even-number-of-digits/1295-find-numbers-with-even-number-of-digits.cpp
class Solution {
public:
    int solve(int n){
        int tmp=n;
        int c=0;
        while(tmp>0){
            c++;
            tmp/=10;
        }
        return c;
    }
    int findNumbers(vector<int>& nums) {
        int c=0;
        for(int i=0;i<nums.size();i++){
            if(solve(nums[i])%2==0){
                c++;
            }
        }
        return c;
    }
};